docs.go

```diff
@@ -1,3 +1,5 @@
+// nolint:gofmt,goimports
```

Comment:
nit: the [`//nolint` directive](https://golangci-lint.run/usage/false-positives/#nolint-directive) shouldn't have a space according to golangci-lint reasoning:
> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.